build-linux.sh

```diff
@@ -107,20 +107,24 @@ run_build()
 
     echo
     echo Building surge-${flavor} with output in build_logs/build_${flavor}.log
+    # Since these are piped we lose status from the tee and get wrong return code so
+    set -o pipefail
```

Comment:
Nice one! Should this be set for the duration of the script though? (same for mac)